dll/win32/msv1_0/usercontext.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * PROJECT:     Authentication Package DLL
+ * LICENSE:     GPL - See COPYING in the top level directory
+ * FILE:        dll/win32/msv1_0/usercontext.c
+ * PURPOSE:     manage user mode contexts (create, destroy, reference)
+ * COPYRIGHT:   Copyright 2019-2020 Andreas Maier <staubim@quantentunnel.de>
+ */
+#ifndef _USERCONTEXT_H_
+#define _USERCONTEXT_H_
+
+#include "precomp.h"
```

Comment:
You include `usercontext.h` inside `precomp.h`. Why so complicated? :)

Reply:
Yes, of course. precomp.h is removed now. Thanks.